static/js/portico/signup.js

```diff
@@ -31,6 +37,13 @@ $(function () {
                 error.insertAfter(element).addClass('help-inline text-error');
             }
         },
+        invalidHandler: function () {
```

Comment:
hm, this is not looking very natural. It's also causing some vertical movement in the layout. Let's remove this for now, and we can deal with it in a separate PR.